mpcenc/mpcenc.h

```diff
@@ -20,6 +20,13 @@
 #ifndef MPCENC_MPCENC_H
 #define MPCENC_MPCENC_H
 
+#ifndef _GNU_SOURCE
+# define _GNU_SOURCE
+#endif
+#ifndef _DEFAULT_SOURCE
+# define _DEFAULT_SOURCE
+#endif
+
```

Comment:
Should this change be replicated in mpcdec, seeing mpcchap?

Reply:
Ah, good catch - very probably

Reply:
It's unclear if it's definitely required, but as a "just in case", we have done so anyway as it won't harm anything to have